### ISSUE TITLE
Interactive arena detection tool fail to display

### ISSUE THREAD

**Issue by caterinalab3** [dattalab/moseq2-app#274]
when I try to set up the parameters like Floor depth range (mm) or Floor mask dilation iterations, Interactive Arena Detection Tool failed to display the background or mouse. It kinds of stack in this page. And I'm using Ubuntu 22.04.3 LTS and azure camera if that could affect the process. I have moseq2-extract version 1.2.0, moseq2-pca version 1.2.0 and moseq2-model version 1.2.0.

[
![Screenshot from 2024-05-17 14-40-30](https://github.com/dattalab/moseq2-app/assets/153322141/5a736f8e-09d1-4409-8803-421a89b27e3f)
](url)

**Comment by gabrielpan147** [comment]
Hello, I am also facing the same issue. Do you find any solutions?

**Comment by BettaDayz** [comment]
I have the same issue, both when using the Docker as well as the Anaconda versions/installs.  I do get a few deprecation warnings in the prior steps before:
/home/thomas/miniconda3/envs/moseq-app/lib/python3.7/site-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.

and

/home/thomas/miniconda3/envs/moseq-app/lib/python3.7/site-packages/sklearn/utils/validation.py:37: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.

but these wouldnt have any influence right?

**Comment by gabrielpan147** [comment]
Yes, I suppose the deprecated lib will not be the case. I initially took it as a RAM issue, but still have not got it fixed.
